docs/introduction.rst

```diff
@@ -1,73 +1,47 @@
 .. include:: defs.rst
 .. _introduction:
 
+********
 Overview
-========
-|AppName| is open source scientific image processing software integrating hardware control, data acquisition, visualization, processing, and analysis using Python. |AppName| is easily extended using Python. It runs on Windows, Linux, and macOS.
+********
+Nion Swift is an image processing software for controlling instruments, and managing the acquisition, organization, visualization, and analysis of data. The software was designed to be used in conjunction with Nion's electron microscopes, but the functionality and the ability to extend its capabilities using Python mean that Nion Swift is useful for any number of data visualization projects. Whether you already have your data, or you want to see the data as it is acquired, Nion Swift helps you visualize your data in exactly the way you want.
```

Comment:
Let's refer to "software" as a plural object, and I still like the initial phrase since it works well in search engines. "Nion Swift is open source scientific image processing software for ..."

Also, I'd back off the "Nion electron microscope" and "extend with Python" only because those are (1) too specific; (2) difficult to explain quickly -- "The software is designed to be used in conjunction with electron microscopes. However, it is also useful in other data visualization applications." (n.b. I'd prefer 'applications' rather than 'projects' to avoid confusion with 'Projects' defined below).

Finally, the last sentence (and several others in this document) still read too sales-y. I would stick more to matter-of-fact sentence structure: "Nion Swift helps you visualize and process your data while it is being acquired and during processing and analysis."